tests/test_equil.cc

```diff
@@ -450,21 +458,21 @@ void test_DeckAllDead()
 
     Opm::EQUIL::DeckDependent::InitialStateComputer<TypeTag> comp(*simulator->problem().materialLawManager(), eclipseState, simulator->vanguard().grid(), 10.0);
     const auto& pressures = comp.press();
-    REQUIRE(pressures.size() == 3);
-    REQUIRE(int(pressures[0].size()) == grid.number_of_cells);
+    BOOST_CHECK(pressures.size() == 3);
+    BOOST_CHECK(int(pressures[0].size()) == grid.number_of_cells);
```

Comment:
These should really be `BOOST_REQUIRE_EQUAL` instead.  The rest of the test is meaningless unless those conditions are satisfied.